### PR TITLE
feature: sanity models for templates

### PR DESCRIPTION
**Pull request by alexvilanovab**

### CODE REVIEW

File: desk-structure.js [hunk 1]
```diff
@@ -49,6 +49,18 @@ export default () =>
...
+        .child(
+          S.document()
+            .schemaType("templates")
+            .documentId("templatesPage")
+            .views([S.view.form()])
+        ),
+      S.listItem()
+        .title("Templates")
+        .child(S.documentList().title("List").filter('_type == "template"')),
+      S.divider(),
```
Comment (andresgutgon) [inline desk-structure.js:63]: What's this?

File: schemas/documents/home.js [hunk 2]
```diff
@@ -62,54 +62,16 @@ export default {
...
-                  { title: "Link button", value: "link" },
-                ],
-              },
-            }),
-            basicSchema({
-              title: "HTML",
-              type: "text",
-            }),
-          ],
-        }),
```
Comment (andresgutgon) [inline schemas/documents/home.js:112]: Is this ok?

File: desk-structure.js (hunk 1 above)
Comment (alexvilanovab) [reply desk-structure.js:63]: this is just a visual divider in the sanity UI. as you can see, there are dividers both above and bellow the templates section

Screenshot 2023-03-07 at 09 56 59

File: schemas/documents/home.js (hunk 2 above)
Comment (alexvilanovab) [reply schemas/documents/home.js:112]: yes, now that we also have videos in the template detail page i had the exact same fields i two places so i decided to make a generic "video" type that we can use in both places. you can see, above the deleted code how it's being used

**Review by geclos — APPROVED** [review]

**Pull request merged**